tests/components/ozw/test_init.py

```diff
@@ -1,18 +1,18 @@
 """Test integration initialization."""
 from homeassistant import config_entries
-from homeassistant.components.zwave_mqtt import DOMAIN, PLATFORMS, const
+from homeassistant.components.ozw import DOMAIN, PLATFORMS, const
 
-from .common import setup_zwave
+from .common import setup_ozw
 
 from tests.common import MockConfigEntry
 
 
 async def test_init_entry(hass, generic_data):
     """Test setting up config entry."""
-    await setup_zwave(hass, fixture=generic_data)
+    await setup_ozw(hass, fixture=generic_data)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)